README.md

```diff
@@ -217,10 +217,14 @@ Next, create an Azure Service Principal, which is a special type of account that
 1. Create a Service Principal with a "contributor" role that is allowed to make changes to any resources in that subscription. Replace $AZURE_SUBSCRIPTION_ID with the ID you found in step 1 and run this command:
 
 ```
-az ad sp create-for-rbac --name "CICD" --role contributor --scopes /subscriptions/$AZURE_SUBSCRIPTION_ID --sdk-auth
+az ad sp create-for-rbac  --sdk-auth --name "github-deployer" --role contributor --scopes /subscriptions/$AZURE_SUBSCRIPTION_ID
```

Comment:
So, the issue blocking my deployment was that there was already a principal with the "CICD" name in the account. I'm guessing that won't happen for the majority of students, but I thought maybe a different name might help? What do you think? I'm still learning how principals work.